doc/reference/tooling/tt_cli/export.rst

```diff
@@ -11,12 +11,17 @@ Exporting data
 
 ..  code-block:: console
 
-    $ tt [crud] export URI SPACE:FILE ... [EXPORT_OPTION ...]
+    $ tt [crud|tdg2] export URI SPACE:FILE ... [EXPORT_OPTION ...]
 
-``tt [crud] export`` exports a space's data to a file.
-The ``crud`` command is optional and can be used to export a cluster's data by using the `CRUD <https://github.com/tarantool/crud>`_ module. Without ``crud``, data is exported using the :ref:`box.space <box_space>` API.
+``tt export`` exports a space's data to a file.
+The ``crud`` and ``tdg2`` commands are optional and cover specific export cases:
```

Comment:
I don't get what "commands are optional" is. There are three different imports, and each of them covers a single specific case:
- `tt import` works **only** with single-replicaset storage (through its master),
- `tt crud import` works **only** with crud sharded cluster (through its router),
- `tt tdg2 import` works **only** with TDG2 sharded cluster (through its router).

It's not like one import command is a subset of another one.